Tools/jit/_stencils.py

```diff
@@ -282,6 +252,7 @@ class StencilGroup:
         default_factory=dict, init=False
     )
     _got: dict[str, int] = dataclasses.field(default_factory=dict, init=False)
+    trampolines: set[int] = dataclasses.field(default_factory=set, init=False)
```

Comment:
Is this accessed from outside of the class itself? If not, let's add an underscore:

```suggestion
    _trampolines: set[int] = dataclasses.field(default_factory=set, init=False)
```

Reply:
Done.